### DISCUSSION TITLE
How does the epicweb-dev/config use eslint 9 already?

### DISCUSSION THREAD

**Discussion by nilsingwersen** [epicweb-dev/epic-stack#743] [Q&A] [answered]
I checked out the epicweb-dev/config repo and I saw that it was already using eslint 9 but it also uses @typescript-eslint 7.10 which is no compatible with eslint 9. How does this work?

**Comment by kentcdodds** [comment, marked as answer]
Because I'm not using their config, just the parser and rules which are compatible

**Reply by nilsingwersen** [reply]
Very interesting, thanks for answering so quick.
